### ISSUE TITLE
Allow `%replaceText%`  and `%deleteText%` to work on content that was imported using `%file()%`

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplates#244]
We often need to import text fragments ("building blocks") that have a generic message but need to include some specific keywords, such as a product name or URL. At the moment the following pattern does not work within the text that is imported:

Template:
```html
  %file("snippets/why-renew.html")%   
  %replaceText("{addonName}","quickFilters")%
  %replaceText("{github}","https://github.com/RealRaven2000/quickFilters")%
```

Contained file (why-renew.html)
```html
<p> 
  All changes of the Add-on {addonName} can be seen as they are 
  created and tracked <a href="{github}/issues">on Github</a>;
</p>
```

At the moment, SmartTemplates does the replacements _before_ the file is loaded into the template. Expected behavior: First load the file and replace its contents into the template, then do the text replacements afterwards.

**Comment by RealRaven2000** [comment]
Preview version with changed behavior:

[smartTemplate-fx-3.17.1pre5.zip](https://github.com/RealRaven2000/SmartTemplates/files/11892386/smartTemplate-fx-3.17.1pre5.zip)

New behavior: SmartTemplates first loads the file and replaces its contents into the template,  then executes the text replacements afterwards. As demonstrated in the example above, it can also be used to modify HTML markup and affect things like HTML attributes such as the URL of a web link.

To install the preview version, as always download the zip file and drag it into Add-ons Manager (do not extract contents).

**Comment by RealRaven2000** [comment]
Implemented in 4.0b3 published on 17/02/2023
